setup.py

```diff
@@ -172,9 +172,18 @@ def additional_commands():
         license='BSD',
         maintainer='ETS Developers',
         maintainer_email='enthought-dev@enthought.com',
-        package_data=dict(traitsui=['extras/images/*', 'image/library/*.zip',
-                                    'images/*', 'wx/images/*',
-                                    'qt4/images/*']),
+        package_data=dict(
+            traitsui=[
+                'examples/demo/*',
+                'examples/demo/*/*',
+                'examples/demo/*/*/*',
+                'examples/demo/*/*',
```

Comment:
`examples/demo/*/*` included twice?

Reply:
i'm not sure what changes need to happen in `MANIFEST.in`

Reply:
> examples/demo/*/* included twice?

Oops. Thanks!

> i'm not sure what changes need to happen in MANIFEST.in

Nothing I think. `MANIFEST.in` already has the top level `examples` folder. We are moving something that is already in the source distribution to the package data (which will also be included in the source distribution).